content/en/opentelemetry/integrations/trace_metrics.md

```diff
@@ -49,15 +49,36 @@ service:
 
 ## Data collected
 
-See [Trace Metrics][2].
+[Trace Metrics][2] are generated for service entry spans and measured spans. These span conventions are unique to Datadog, so OpenTelemetry spans are identified with the following mapping:
+| OpenTelemetry Convention | Datadog Convention |
+| --- | --- |
+| Root span | Service entry span |
+| Server span (`span.kind: server`) | Service entry span |
+| Consumer span (`span.kind: consumer`) | Service entry span |
+| Client span (`span.kind: client`) | Measured span |
+| Producer span (`span.kind: producer`) | Measured span |
+| Internal span (`span.kind: internal`) | No trace metrics generated |
 
-## Full example configuration
+[`SpanKind`][3] is typically set when a span is created, but can also be updated by using the [transform processor][4] to control the mapping above. For example, if trace metrics are desired for an internal span, the following configuration transforms an internal span with `http.path: "/health"` into a server span:
```

Comment:
nit: this example actually marks the internal span as service entry span, not just span with trace metrics

Reply:
Just updated to transform into client span instead